bcl2fastq/index_summary.py

```diff
@@ -110,18 +143,24 @@ def update_pruned_unknown_hidden(run_alias):
     [dep.Input('sample_run_hidden', 'children')]
 )
 def update_known_index_bar(run_json):
+    """ When input (sample_run_hidden) is changed, function is rerun to get update bar graph of sample indices
+
+           Attributes:
+               run(list): name of run
+               data(list): empty list appended with values from "For" function
+           Parameters:
+               run_json(list): json format of run variable
```

Comment:
probably a `str` format instead of a `list`, and it's probably more accurate to call it `run data`; calling it `run variable` makes it sound like `run_json` depends on `run` existing already, when the opposite is true. (Same as method below)